debezium-core/src/main/java/io/debezium/config/CommonConnectorConfig.java

```diff
@@ -54,7 +54,6 @@
  */
 public abstract class CommonConnectorConfig {
     public static final String TASK_ID = "task.id";
-    public static final String LOGICAL_NAME = "logical.name";
     public static final String LOGIC_NAME_PLACEHOLDER = "${logical.name}";
```

Comment:
`LOGIC_NAME_PLACEHOLDER` is now unused and should be removed (IIRC you had it in previous version - it was fine, I just wanted to fix the description not to refer to MySQL specifically the the config field is used also in other connectors).

Reply:
done.